content/200-concepts/100-components/03-prisma-migrate/index.mdx

```diff
@@ -263,14 +263,14 @@ The `migrate dev` command will prompt you to reset the database in the following
 - Migration history conflicts caused by [modified or missing migrations](#do-not-edit-or-delete-migrations-that-have-been-applied)
 - The database schema has drifted away from the end-state of the migration history
 
+### Reset the development database
+
 You can also `reset` the database yourself to undo manual changes or `db push` experiments by running:
```

Comment:
Unrelated to your changes: I don't understand that part `or `db push` experiments by running:`

Reply:
Maybe it would make more sense with `db push --force-reset`, not sure if it helps though

Reply:
For me generally teh docs do not work that are like a story, where you have to have read everything before. Seems like this would still be the case with that change.